main.cpp

```diff
@@ -545,11 +431,125 @@ MOVE THEM to the space below this block comment and put them in numerical order
     simply CUT and PASTE them in the space provided below:
 */
 
+Thing 1) Supermarket
+5 properties:
+    1)Total sales in a day (double)
+    2)electricity consumed per month (float)
+    3)number of lights (int)
+    4)number of fans (int)
+    5)number of products available (double)
+3 things it can do:
+    1)sell products
+    2)charge customers
+    3)home deliver products
 
+Thing 2) School
+5 properties:
+    1)number of students in a class (int)
+    2)number of male students (double)
+    3)number of female students (double)
+    4)name of the school (std::string)
+    5)water consumption per day (float)
+3 things it can do:
+    1)teach
+    2)collect fee from students
+    3)conduct exams
 
+Thing 3) Book store
+5 properties:
+    1)number of fans (int)
+    2)number of books (double)
+    3)genere of books (std::string)
+    4)books that can be rented (std::string)
+    5)customers per day (int)
+3 things it can do:
+    1)collect charges on a monthly basis
+    2)restock books
+    3)offer discounts
 
+Thing 4) Zoo
+5 properties:
+    1)variety of species (std::string)
+    2)number of visitors per day (int)
+    3)number of restrooms (int)
+    4)number of foodstalls (int)
+    5)area of the zoo (float)
+3 things it can do:
+    1)charge visitors
+    2)provide visitors a safari
+    3)guide the visitors
 
+Thing 5) Laptop
+5 properties:
+    1) Display
+    2) Memory
+    3) Battery
+    4) Applications
+    5) I/O
+3 things it can do:
+    1) automate tasks
+    2) send email
+    3) run applications
```

Comment:
This is supposed to be thing 10.
Thing 6 is supposed to be Thing 5.
Thing 7 is supposed to be Thing 6
Thing 8 is supposed to be Thing 7
Thing 9 is supposed to be Thing 8
Thing 10 is supposed to be Thing 9

and all of these Plain-English descriptions are supposed to be commented out so that no errors are produced when you run your project. 

- [ ] revise